azure-pipelines.yml

```diff
@@ -33,7 +33,7 @@ variables:
   disable_test_parallelization: true
   DOTNET_CLI_TELEMETRY_OPTOUT: 1
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
-  dotnet_core_version: 5.0.x
+  dotnet_core_version: 6.0.x
```

Comment:
Minor: Update the name in line 42 from `build_net5` to `build_net6`

Reply:
Good catch, fixed in https://github.com/JasperFx/marten/pull/1977/commits/60a5558689ae7a42e916b3c8d60aa000b635d45d.